application/backend/src/Mailer/MailSenderInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Mailer;
+
+interface MailSenderInterface
```

Comment:
What's the point of this interface over putting the `send()` method directly into the abstract class ?